src/navigation/AppNavigation.js

```diff
@@ -1,14 +1,16 @@
 import {createSwitchNavigator} from 'react-navigation';
 import HomeScreen from '../screens/homeScreen';
 import ActivityDayScreen from '../screens/activityDayScreen';
+import AccountScreen from '../screens/accountCreationScreen';
```

Comment:
Best practice to name the import after what you're importing i.e. 
```suggestion
import AccountCreationScreen from '../screens/accountCreationScreen';
```

Reply:
Yeah I think actually that should just be "accountScreen" since I would assume we'll use the same one to update the account